docs/docs/graphql-reference.md

```diff
@@ -58,6 +58,10 @@ Dates can be formatted using the `formatString` function.
 
 <iframe src="https://gatsbygraphql.sloppy.zone/?query=%7B%0A%20%20allMarkdownRemark(%0A%20%20%20%20filter%3A%20%7Bfrontmatter%3A%20%7Bdate%3A%20%7Bne%3A%20null%7D%7D%7D%0A%20%20)%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20frontmatter%20%7B%0A%20%20%20%20%20%20%20%20%20%20title%0A%20%20%20%20%20%20%20%20%20%20date(formatString%3A%20%22dddd%20DD%20MMMM%20YYYY%22)%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D" width="600" height="400"></iframe>
 
+Gatsby relies on [Moment.js](https://momentjs.com/) to format the dates. This allows you to use any tokens in your string. See [moment.js documentation](https://momentjs.com/docs/#/displaying/format/) for more tokens.
+
+Example: `anotherDate(formatString: "dddd, MMMM Do YYYY, h:mm:ss a") // Sunday, August 5th 2018, 10:56:14 am`
```

Comment:
This could be a link instead of an iframe? Then people can either copy & paste into one of the existing iframes or click the link to edit it directly.

Reply:
What do you mean? A link to Moment.js’s documentation?

Reply:
Sorry I didn't explain that very clearly! I added another commit to show what I meant

Reply:
Oh ok I see. Good catch, thank you :)